### ISSUE TITLE
Other Brand Filament

### ISSUE THREAD

**Issue by joshs85** [ANYCUBIC-3D/AnycubicSlicer#15]
The slicer only has options for Anycubic filament.  Please add profiles for other brands.

**Comment by nolantern** [comment]
You can add as much user presets as you want.
Just edit an existing filament and save it with a new name. Keep in mind, that the new filament inherits all unchanged settings from the source filament. Altering attributes of the source filament changes the child filament too.

Alternatively you could import filaments created in orca slicer. Or you export some filament, edit the text file and import it as a new filament.

**Comment by joshs85** [comment]
Right.  This is the problem.  They should be there already.  I don't want to have to add them myself.
